torch/csrc/jit/import_export_helpers.cpp

```diff
@@ -0,0 +1,42 @@
+#include "jit/import_export_helpers.h"
+
+#include <algorithm>
+
+namespace torch {
+namespace jit {
+namespace ImportExportHelpers {
```

Comment:
Do we really need a new namespace? Its style doesn't match namespace style.

Reply:
I am basically trying to accomplish the same thing as a class with only static methods.